index.js

```diff
@@ -130,21 +112,17 @@ function createWrapFn(originalsToProxies, proxiesToOriginals, unwrapFn) {
 /**
  * @param {any} obj
  */
-function membrane(obj) {
-    const originalProxies = new WeakMap();
-    const originalTargets = new WeakMap();
-    const outerProxies = new WeakMap();
-
-    const wrap = createWrapFn(originalProxies, originalTargets, unwrap);
-    const wrapOuter = createWrapFn(outerProxies, originalProxies, wrap)
+function membrane(left, right) {

```

Comment:
Such approach seems to be not an original `Membrane`, since in time of creation we have to be aware of both membrane sides.
I'm not sure that such changed pattern covers all the original use-cases. Am I mistaken?